Tests/OpenCombineTests/PublisherTests/DecodeTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  DecodeTests.swift
+//  
+//
+//  Created by Joseph Spadafora on 6/21/19.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class DecodeTests: XCTestCase {
+    static let allTests = [
+        ("testDecodeWorks", testDecodeWorks)
+    ]
+    
+    private let jsonEncoder = JSONEncoder()
+    private let jsonDecoder = JSONDecoder()
+    
+    var cancel: Cancellable?
+    
+    func testDecodeWorks() {
+        let promise = XCTestExpectation(description: "decode")
```

Comment:
Since everything should happen synchronously, I think it'd be better not to use the `XCTestExpectation ` API, and go with the synchronous testing instead.

In this case, that means just deleting this line and the `wait(for: [promise], timeout: 1)` line.